free-1.md

```diff
@@ -6,63 +6,92 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
+Currently, it is difficult for a non-technical user to create a personal
+website with enough flexibility. Also, it is difficult for such a user to 
+understand the concepts behind how websites work such as the front-end and
+back-end. The DSL would be typed in Microsoft Word or Powerpoint and be able to 
+hook in and Excel spreadsheet essentially creating a dynamic website.
 
 
 ### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
+I think a DSL would be a good fit for this because Word documents are
+similar to web framework templates which use a DSL. A DSL would allow
+users to work in a familiar environment (MS Office) with just a few changes
+in syntax. 
+
 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
+I am excited about this idea because it could bring website creation to 
+another user base of non-technical users. Originally it was a joke that 
+some of my coworkers over the summer had. The joke was that people with 
+no development experience would create a website with Microsoft Office. 
+Word and Powerpoint would be the front-end and Excel would be the back-end.
+As the saying goes, "There is a grain of truth in every joke."
 
 
 ### Domain
-_Describe the project's domain in five words._
+Website creation with Microsoft Office
 
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
+I imagine the interface being something like web templates for popular 
+frameworks such as Django. In Django, you can reference a variable from
+the backend with the syntaxt {{ var_name }}. I could imagine changing
+this to reference the row/col in an excel file e.g. {{ A4 }}.
 
 
 ### Operation (semantics)
-_What might happen when a program runs? How does a program interact with the
-user? What kinds of errors might occur, and how might they be communicated to
-the user?_
+The program would take in Word documents and Excel spreadsheets as input
+and create a html website as output. There could be many possible errors
+that would need to be communicated to the user such as syntax errors or
+file_not_found errors. Perhaps I could create an error checker for this.
 
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+It should be easy to create a simple dynamic web page with the language. 
```

Comment:
Dynamic-ness in the usual HTML framework generally requires at least a bit of JavaScript. Was the idea that parts of your DSL would turn into HTML and parts would turn into JS?